src/org/ods/component/CopyImageStage.groovy

```diff
@@ -90,22 +88,24 @@ class CopyImageStage extends Stage {
         }
     }
 
-    private int copyImage(sourcetoken, targetInternalRegistryToken, String dockerProtocol, String copyparams) {
+    private int copyImage(sourcetoken, targettoken, String dockerProtocol, String copyparams) {
         def registryPath = this.options.repo ? \
            "${this.options.registry}/${this.options.repo}/${this.options.image}" : \
            "${this.options.registry}/${this.options.image}"
+
         int status = steps.sh(
             script: """
                 skopeo copy ${copyparams} \
                 --src-tls-verify=${this.options.verifyTLS} ${sourcetoken} \
                 ${registryPath} \
-                --dest-creds openshift:${targetInternalRegistryToken} \
+                ${targettoken} \
```

Comment:
Passing credentials in the command line is insecure, as they will be logged in more than one place. We should consider refactoring this code to use a secret that we can map as the auth.json file that Skopeo can use. The same should be done with the source credentials. However, the refactoring isn't in the scope of this PR, and we aren't adding a new vulnerability, so this is not a blocker.